config/jobs-chromeos.yaml

```
@@ -350,6 +350,11 @@ jobs:
   baseline-x86-stoneyridge: *baseline-job
   baseline-x86-stoneyridge-staging: *baseline-job
 
+  kbuild-gcc-10-arm64-chromebook:
+    <<: *kbuild-gcc-10-arm64-job
+    fragments:
+      - arm64-chromebook
```

```suggestion
  kbuild-gcc-10-arm64-chromebook:
    <<: *kbuild-gcc-10-arm64-chromeos-job
    params:
      <<: *kbuild-gcc-10-arm64-chromeos-params
      cross_compile_compat:
      defconfig: defconfig
```

`kbuild-gcc-10-arm64-job` is only defined in `pipeline.yaml` (at least for now), and having it referenced in a different file causes parsing issues as cross-files anchors unfortunately aren't supported. jobs-chromeos.yaml does have the `kbuild-gcc-10-arm64-chromeos-job` (and corresponding `-params`) aliases though, so you can use those.

`fragments` already contain the proper value (unless you want/need to get rid of `CONFIG_MODULE_COMPRESS=n`), but `defconfig` should be set back to the default upstream defconfig, and I have a feeling (which might be wrong) `cross_compile_compat` should be reset to null in this case.